src/index.tsx

```diff
@@ -112,10 +120,10 @@ export default class DraggableList<
     heights: null,
   };
 
-  public getItemInstance(key: string): T {
```

Comment:
Is the type signature still meant to be the same? I'd prefer to keep it explicit here so we get an error if the new body isn't keeping the contract.

Reply:
[0e302f8](https://github.com/StreakYC/react-draggable-list/pull/62/commits/0e302f89df55444b546bc56818cbc0688ec230bf)